backend/src/main/resources/application.yml

```diff
@@ -3,6 +3,16 @@ spring:
         name: jobwait
     datasource:
         driver-class-name: org.postgresql.Driver
-        url: jdbc:postgresql://localhost:5432/mydatabase
+        url: jdbc:postgresql://localhost:5432/test
```

Comment:
does this work? i think the name has to be the same as the name in the [compose.yaml](https://github.com/pecasdev/jobwait/blob/882725dca98f46dc809d48cdfe53b3ccb6ad6217/backend/compose.yaml#L8)

Reply:
sorry ill remove this was from really early stuff when i was trying to get flyway to work.